src/z3c/sqlalchemy/base.py

```diff
@@ -140,3 +140,4 @@ def _createEngine(self):
                                             autoflush=True,
                                             **self.session_options))
         register(self._sessionmaker)
+        self._session = self._sessionmaker()
```

Comment:
For the same reason above I found convenient to persist the session after creating the engine